### ISSUE TITLE
Add code snippets section to landing page

### ISSUE THREAD

**Issue by rihp** [Web3-API/dao#79]
## Inspiration:

https://amberdata.io/
![image](https://user-images.githubusercontent.com/12145726/103799388-5001da00-504b-11eb-852e-e576f28f0629.png)

**Comment by rihp** [comment]
Drafted something, maybe we can iterate around this to merge a good snippet soon. The text can be copied and edited on the browser.
https://github.com/Web3-API/landing-page/tree/code-snippets

![image](https://user-images.githubusercontent.com/12145726/104066304-7cf7ed80-5201-11eb-8f79-946d0e1566f3.png)

I'd like to make it so that the box has round edges, and with a smaller width.

**Comment by rihp** [comment]
Updated view : 
![image](https://user-images.githubusercontent.com/12145726/104137742-5c659a00-539f-11eb-8ba1-ba2abc4e94ab.png)

**Comment by dOrgJelli** [comment]
Now that we have the developer docs which include code snippets, I think we can hold off on this to make sure the landing page is as simple and high level as possible. Just my opinion, do you have a differing one?

**Comment by kev-ngo** [comment]
That looks 🔥  @rihp . 

Maybe we can get the rebrand team's thoughts once they work on the new landing page.

**Comment by rihp** [comment]
agreed with kevin, no need to push this as a current priority, but would be nice to revisit in about a month
